libs/openFrameworks/communication/ofArduino.h

```diff
@@ -458,31 +460,31 @@ class ofArduino{
 	list<int> _digitalHistory[ARD_TOTAL_DIGITAL_PINS];
 	// a history of received data for each digital pin
 	
-	int _digitalPinMode[ARD_TOTAL_DIGITAL_PINS];
+	int _digitalPinMode[ARD_TOTAL_DIGITAL_PINS]={INT_MAX};
 	// the modes for all digital pins
 	
-	int _digitalPinValue[ARD_TOTAL_DIGITAL_PINS];
+	int _digitalPinValue[ARD_TOTAL_DIGITAL_PINS]={INT_MAX};
 	// the last set values (DIGITAL/PWM) on all digital pins
 	
-	int _digitalPortValue[ARD_TOTAL_PORTS];
+	int _digitalPortValue[ARD_TOTAL_PORTS]={INT_MAX};
 	// the last set values on all ports
 	
-	int _digitalPortReporting[ARD_TOTAL_PORTS];
+	int _digitalPortReporting[ARD_TOTAL_PORTS]={INT_MAX};
 	// whether pin reporting is enabled / disabled
 	
-	int _digitalPinReporting[ARD_TOTAL_DIGITAL_PINS];
+	int _digitalPinReporting[ARD_TOTAL_DIGITAL_PINS]={INT_MAX};
 	// whether pin reporting is enabled / disabled
 	
-	int _analogPinReporting[ARD_TOTAL_ANALOG_PINS];
+	int _analogPinReporting[ARD_TOTAL_ANALOG_PINS]={INT_MAX};
 	// whether pin reporting is enabled / disabled
 	
 	bool bUseDelay;
 	
-	bool connected;
+	bool connected;  // true if serial connection to Arduino exists
```

Comment:
@bakercp I added the doxygen commands for *my* changes. I would leave the rest to when someone (I?) goes over the whole class and does style fixes etc.